pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -522,6 +544,46 @@ func (h *NetworkUtilsHandler) DisableTXOffloadChecksum(ifaceName string) error {
 	return nil
 }
 
+func (h *NetworkUtilsHandler) CreateRADaemon(socketPath string, advertisementIfaceName string, ipv6CIDR string, currentRetry int) error {
+	c, err := net.Dial("unix", socketPath)
+	if err != nil {
+		log.Log.V(4).Warningf("could not connect to the unix domain socket: %v", err)
+
+		if currentRetry > 0 {
+			time.Sleep(time.Second)
+			return h.CreateRADaemon(socketPath, advertisementIfaceName, ipv6CIDR, currentRetry-1)
+		}
+	}
+	defer c.Close()
+	fdConn := c.(*net.UnixConn)
+
+	log.Log.Info("managed to connect to unix domain socket; will retrieve the opened FD from it")
+
+	expectedFDNumber := 1
```

Comment:
please change to `expectedFDsNumber`. The existing name confused me and I thought at first glance it is the number representation of the desired fd.

Reply:
https://github.com/kubevirt/kubevirt/pull/4581/commits/26cc20c74a84772955c5b5a4b5f25451d94b0f53#diff-c14df878b1a4600e0af398a09886ec253fee24719d7d30b87ff8e73b1810b39aR120